spec/abstractops.html

```diff
@@ -1442,4 +1442,190 @@ <h1>ParseTemporalYearMonthString ( _isoString_ )</h1>
         }.
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-temporal-topositiveintegerorinfinity" aoid="ToPositiveIntegerOrInfinity">
+    <h1>ToPositiveIntegerOrInfinity ( _argument_ )</h1>
+    <p>
+      The abstract operation ToPositiveIntegerOrInfinity takes argument _argument_. It converts _argument_ to +&infin; or an integer _i_ for which <emu-eqn>0 &lt; i</emu-eqn>. It performs the following steps:
+    </p>
+    <emu-alg>
+      1. Let _integer_ be ? ToInteger(_argument_).
+      1. If _integer_ is *-&infin;*<sub>𝔽</sub>, then
+        1. Throw a *RangeError* exception.
+      1. If _integer_ &le; 0, then
+        1. Throw a *RangeError* exception.
+      1. Return _integer_.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-temporal-preparetemporalfields" aoid="PrepareTemporalFields">
+    <h1>PrepareTemporalFields ( _temporalLike_, _fieldNames_, _optionalFieldNames_ )</h1>
```

Comment:
nit: _temporalLike_ seems to imply that "a Temporal" is a thing. Maybe _fieldsObject_ would be a better name?